robot/robot.go

```diff
@@ -80,6 +81,12 @@ import (
 //
 //	// Cancel all current and outstanding operations for the machine and stop all actuators and movement.
 //	err := machine.StopAll(ctx)
+//
+// Shutdown example:
+//
+// 	// Shut down the robot.
+//	err := machine.Shutdown(ctx)
```

Comment:
Would this and above not also have to have ctx declared?

Reply:
actually I don't think it needs this